reference/artifacts/in-pipelines/index.md

```diff
@@ -11,7 +11,7 @@ sidebar:
 > history in Redis. Deleting recent executions from Redis can cause
 > unexpected behavior.
 
-Now that you have an idea of [what an artifact is](/reference/artifacts#about-spinnaker-artifacts) in Spinnaker, you need to
+Now that you have an idea of [what an artifact is](https://www.spinnaker.io/reference/artifacts/#about-spinnaker-artifacts) in Spinnaker, you need to
```

Comment:
I don't think this is the right approach, since this breaks if we ever move the hostname.